src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const pictures = document.querySelectorAll('.list-item__link');
+const valide = document.getElementById('largeImg');
```

Comment:
Use a more descriptive variable name for the 'valide' variable, such as 'mainImage'.